File: Client/src/app/Restrito/cad-servicos/cad-servicosVM/cadastra-servico-view-model.ts
import { VantagemViewModel } from './vantagem-view-model';
import { ImagemViewModel } from './imagem-view-model';
// import { SafeUrl, SafeResourceUrl } from '@angular/platform-browser';

export class CadastraServicoViewModel {
    id: number;
    iconFont: string;
    titulo: string;
    urlVideo: string;
    imagem: ImagemViewModel;
    descricaoCurta: string;
    descricaoLonga: string;
    listaVantagens: VantagemViewModel[];
    ativo: boolean;

    constructor() {
        this.iconFont = '';
        this.titulo = '';
        this.urlVideo = '';
        this.imagem = new ImagemViewModel();
        this.descricaoCurta = '';
        this.descricaoLonga = '';
        this.listaVantagens = new Array<VantagemViewModel>();
        this.ativo = true;
    }
}

File: ViewModels/Servicos/CadastraServicoViewModel.cs
using compasso.Models;
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Threading.Tasks;

namespace compasso.ViewModels.Servicos
{
    public class CadastraServicoViewModel
    {
        public int Id { get; set; }
        public string Titulo { get; set; }
        public string UrlVideo { get; set; }
        public ImagemViewModel Imagem { get; set; }
        public string DescricaoCurta { get; set; }
        public string DescricaoLonga { get; set; }
        public bool Ativo { get; set; }
        public ICollection<VantagemViewModel> ListaVantagens { get; set; }


        public CadastraServicoViewModel()
        {
            this.ListaVantagens = new List<VantagemViewModel>();
            this.Imagem = new ImagemViewModel();
        }
    }
}
